src/libstd/sys/unix/ext/fs.rs

```diff
@@ -20,7 +20,9 @@ use sys;
 use sys_common::{FromInner, AsInner, AsInnerMut};
 use sys::platform::fs::MetadataExt as UnixMetadataExt;
 
-/// Unix-specific extensions to `File`
+/// Unix-specific extensions to [`File`].
+///
+/// [`File`]: ../../../std/fs/struct.File.html
```

Comment:
There are still broken links (not enough `../`)

```
[01:12:39] std/os/unix/fs/trait.DirEntryExt.html:54: broken link - std/std/fs/struct.DirEntry.html

[01:12:39] std/os/unix/fs/index.html:59: broken link - std/std/fs/struct.DirBuilder.html

[01:12:39] std/os/unix/fs/index.html:66: broken link - std/std/fs/struct.DirEntry.html

[01:12:39] std/os/unix/fs/index.html:73: broken link - std/std/fs/struct.File.html

[01:12:39] std/os/unix/fs/trait.FileExt.html:55: broken link - std/std/fs/struct.File.html

[01:12:39] std/os/unix/fs/trait.DirBuilderExt.html:54: broken link - std/std/fs/struct.DirBuilder.html
```

Reply:
Forgot a `..` once again...